#Camera Info
uint32 cameraZMax
uint32 cameraXMax
float32 cameraYOffset

#Occupancy Grid Info
float32 gridResolution
uint32 gridCameraZ
uint32 gridCameraX

